src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,13 +672,13 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         Error(diagnostics, ErrorCode.ERR_AnonymousTypeDuplicatePropertyName, argumentSyntax.NameColon.Name);
```

Comment:
@jcouv I am not sure what do you mean saying:
> One is about specifying all names

The second error is about uniqueness. I don't believe it is helpful to report this error if we already said the name is invalid due to the other reason.